client/state/account-recovery/test/reducer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+} from 'state/action-types';
+
+import dummyData from './test-data';
+import reducer from '../reducer';
+
+describe( 'account-recovery reducer', () => {
+	const camelDummyData = {
+		email: dummyData.email,
+		emailValidated: dummyData.email_validated,
+		phone: dummyData.phone,
+		phoneValidated: dummyData.phone_validated,
+	};
+
+	it( 'should return an initial object with the settings data.', () => {
+		const initState = reducer( null, {
+			type: ACCOUNT_RECOVERY_FETCH_SUCCESS,
+			accountRecoverySettings: dummyData,
+		} );
+
+		assert.deepEqual( initState, camelDummyData );
+	} );
+
+	it( 'should return a new state object with the settings data.', () => {
+		const prevState = {
+			foo: '1',
+			bar: 'bar',
+		};
```

Comment:
Not so necessary when using the `createReducer` helper, but it can be helpful to use `deepFreeze` in tests to ensure the state isn't mutated:

```js
import deepFreeze from 'deep-freeze';

const prevState = deepFreeze( { foo: '1', ... } );
```